src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -669,8 +671,8 @@ public void WindowsSearchPatternInvalid_Wildcards_netcoreapp()
         [PlatformSpecific(TestPlatforms.AnyUnix)]  // Unix-invalid search patterns throws no exception 
```

Comment:
The comment for `WindowsSearchPatternInvalid_Wildcards_netcoreapp` isn't right anymore. Comment should be `\\ .NET Core doesn't throw on wildcards`. .NET Framework did: https://referencesource.microsoft.com/#mscorlib/system/io/filesystemenumerable.cs,232

We can also make this cross plat (i.e. remove `[PlatformSpecific]`).